<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RicercaCliente</class>
 <widget class="QWidget" name="RicercaCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ricerca Cliente</string>
  </property>
  <widget class="QListView" name="lista_clienti">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>531</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci ID Cliente:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_barra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cerca">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apri">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apri</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
